docs/concept-exercises.md

```diff
@@ -49,9 +49,9 @@ When designing Concept Exercises, please consider the following guidelines:
 
 When writing documentation and supporting Markdown files for Concept Exercises, please consult the [style guide][style-guide] noting the [automatic formatting section][style-guide-auto-formatting]. Also check any language-specific style guides, where applicable.
 
-## Exercise Structure
+## Exercise-specific files
```

Comment:
Suggestion that we link to the YouTube video here?

Reply:
Updated.